website/docs/components/text/partials/code/how-to-use.md

```diff
@@ -122,8 +135,13 @@ We can imagine even more complex examples where we may want to apply a typograph
 <Hds::Text::Body @tag="div" @size="200" @color="strong">
   <p>This is some generic text in a paragraph.</p>
   <ul>
-    <li>this list item contains some <strong>strong</strong> and <em>em</em> tags</li>
-    <li>this list item contains a <Hds::Link::Inline @href="#">link</Hds::Link::Inline></li>
+    <li>this list item contains some
```

Comment:
[blocker?] these code-reformatting are impacting the code snippets shown to the readers/consumers. I know it's a pain, but could it be possible to undo the changes? 

(my 2cents tip: I try to avoid running the automatic reformatter on these markdown pages, exactly for this reason; even if it's _technically_ code, at the end of the day it's markdown, it just have to work, for me)

Reply:
Yeah, thanks for catching this, I need to figure out what is happening with my Prettier settings that are overriding these things. I'll revert them

Reply:
Updated, those settings are pretty tricky to find, I did remove some config that I had so should be good going forward on this.